### PR TITLE
ENH: add kraken2-inspect action

### PR DESCRIPTION
**Pull request by colinvwood**

### CODE REVIEW

**Comment by colinvwood** [comment]
blocked by bokulich-lab/q2-types-genomics#62

File: q2_moshpit/plugin_setup.py [hunk 1]
```diff
@@ -287,6 +287,29 @@
     citations=[citations["wood2019"], citations["lu2017"]],
 )
 
+plugin.methods.register_function(
+    function=q2_moshpit.kraken2.database.inspect_kraken_db,
```
Comment (misialq) [inline q2_moshpit/plugin_setup.py:291]: Could we maybe change it to `inspect_kraken2_db`? Just to be explicit about which Kraken version we're using.

**Review by misialq — COMMENTED** [review]
Looks great, thanks @colinvwood! Approval waiting for the CI to pass 😆

**Comment by gregcaporaso** [comment]
I don't need to review this one if @misialq is reviewing, unless you all specifically want my feedback.

**Review by misialq — APPROVED** [review]
Hey @colinvwood, LGTM - thanks! 🙏  I'm merging this one.

**Pull request merged**